src/client/pages/Download/components/CrossSells.tsx

```diff
@@ -42,6 +44,13 @@ export const CrossSells = () => {
   const { crossSells } = useActiveContractBundles()
   const textKeys = useTextKeys()
   const currentLocale = useCurrentLocale()
+  const [trackEvent] = useTrackEvent('cross_sell_click')
+
+  const handleOnClick = (type: CrossSellType) => {
```

Comment:
Why extra wrapper that does not add value?

You can add `useCallback` or directly use `onClick={() => trackEven(crossSell.type)}`

Reply:
Yeah, you're right! I did some debugging in that function before, but removed it before the PR and now it makes no sense to have that extra code. Will remove it 👍